_security/configuration/yaml.md

```diff
@@ -249,7 +246,7 @@ requests: # Only allow GET requests to /sample-index1/_doc/1 and /sample-index2/
 
 ## roles.yml
 
-This file contains any initial roles that you want to add to the Security plugin. Aside from some metadata, the default file is empty, because the Security plugin has a number of static roles that it adds automatically.
+Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/roles.yml`, this file contains any initial roles that you want to add to the Security plugin. Aside from some metadata, the default file is empty, because the Security plugin has a number of static roles that it adds automatically.
```

Comment:
```suggestion
This file contains any initial roles that you want to add to the Security plugin. Aside from some metadata, the default file is empty, because the Security plugin has a number of static roles that it adds automatically. You can find the `roles.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/roles.yml`.
```